application/src/main.rs

```diff
@@ -262,105 +252,49 @@ mod app {
         )
     }
 
-    #[task(binds = USART1, priority=1, local = [usart_rx, to_host_serial], shared = [shell_status, led_rx])]
+    #[task(binds = USART1, priority=1, local = [usart_rx], shared = [led_rx, serial2])]
     fn usart_task(cx: usart_task::Context){
         let usart_rx = cx.local.usart_rx;
-        let shell_status = cx.shared.shell_status;
-        let led_rx = cx.shared.led_rx;
-        let to_host_serial = cx.local.to_host_serial;
+        let led_rx   = cx.shared.led_rx;
+        let serial2  = cx.shared.serial2;
```

Comment:
may be now we should just call it dut_usb_serial or dut_vcp? there is no serial1